bosh_cli/lib/cli/client/director.rb

```diff
@@ -554,6 +554,36 @@ def fetch_backup
           path
         end
 
+        def restore_db(filename, options={})
```

Comment:
We also noticed that although options are allowed here, they are not passed in either existing use case, always defaulting to an empty hash.